<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindowView</class>
 <widget class="QDialog" name="SettingsWindowView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>338</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>701</width>
     <height>311</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="SlideShowSettings">
    <attribute name="title">
     <string>Slideshow Settings</string>
    </attribute>
    <widget class="QDoubleSpinBox" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>120</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>230</y>
       <width>341</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>281</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Slideshow Timer Interval (seconds)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>271</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Info Timer Interval (seconds)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Slide Folder Location</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>20</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Disable Fullscreen Mode</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_5">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>160</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Begin on Info Slide</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>230</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>191</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Default</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Alphabetical</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Reverse Alphabetical</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Randomized</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>170</y>
       <width>191</width>
       <height>28</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Default</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Alphabetical</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Reverse Alphabetical</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Randomized (incompatible with consecutive info slides</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Content Slide Play Order</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Info Slide Play Order</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_6">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>90</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Consecutive Info Slides</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>checkBox_5</zorder>
    <zorder>lineEdit</zorder>
    <zorder>lineEdit_2</zorder>
    <zorder>lineEdit_4</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>label_4</zorder>
    <zorder>checkBox</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>comboBox</zorder>
    <zorder>comboBox_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_5</zorder>
    <zorder>checkBox_6</zorder>
   </widget>
   <widget class="QWidget" name="ImageFolderSettings">
    <attribute name="title">
     <string>Manage Slide Folders</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>681</width>
       <height>311</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Folder Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Include in Slideshow</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Delete from Local Storage</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>434</x>
     <y>340</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
